ArduPlane/tiltrotor.cpp

```diff
@@ -271,12 +271,16 @@ void QuadPlane::tilt_compensate_down(float *thrust, uint8_t num_motors)
     }
 
     float largest_tilted = 0;
+    const float sin_tilt = sinf(radians(tilt.current_tilt*90));
 
     // now constrain and apply _tilt_equal_thrust if enabled
     for (uint8_t i=0; i<num_motors; i++) {
         if (is_motor_tilting(i)) {
             if (equal_thrust) {
                 thrust[i] = tilt_total / tilt_count;
+                // add in differential thrust for yaw control, scaled by tilt angle
+                const float diff_thrust = motors->get_roll_factor(i) * motors->get_yaw() * sin_tilt;
```

Comment:
We could reused the current plane mode rudder as scaled by `RUDD_DT_GAIN`, that would be equivalent to what is done in forward flight.
```suggestion
                const float diff_thrust = motors->get_roll_factor(i) * sin_tilt * plane.rudder_dt;
```
Or apply `RUDD_DT_GAIN` to this directly to the copter controller gain. I suspect using the copter controller rather than plane means that you have greatly increased yaw authority and might get osculations.

```suggestion
                const float diff_thrust = motors->get_roll_factor(i) * motors->get_yaw() * sin_tilt *  plane.g2.rudd_dt_gain.get() * 0.01f;
```

Reply:
https://www.merriam-webster.com/dictionary/osculation

Reply:
I tried to use motors->get_yaw() and it failed, but the simulation was fine.
Maybe try using transition_yaw_target=wrap_PI(plane.ahrs.yaw)

https://youtu.be/Da3_XH7_orc

Reply:
using initial transition yaw should work, although we'd need to "unlock" it if the pilot is giving input, like we do for CRUISE mode